MetamodelAlgemeen.md

```diff
@@ -1586,24 +1588,68 @@ Een objecttype gebruikt attributen voor het specificeren van eigenschappen.
 Een objecttype gebruikt gegevensgroepen voor het specificeren van groepen van
 eigenschappen.
 
-*Toepassing*: objecttypen met gegevensgroepen. Of een gegevensgroeptype dat zelf ook
+*Toepassing*: Objecttypen met gegevensgroepen of een gegevensgroeptype dat zelf ook
 weer een gegevensgroeptype bevat.
 
-#### Metagegeven: **Gegevensgroeptype**
+#### Metagegeven: **heeft gegevensgroeptype**
 
->   **Definitie Gegevensgroeptype**  
+>   **Definitie heeft gegevensgroeptype**  
```

Comment:
Ik lees hier "Definitie heeft gegevensgroep" als een zin. Met aanhalingstekens of cursief wordt het duidelijker wat bedoeld wordt.  Definitie "heeft gegevensgroep" of Definitie _heeft gegevensgroep_

Reply:
Zag ik ook. Moeten we iets op bedenken. Je zou een format willen hebben voor het duiden van modelelementen. Tussen '' of een kleur. NEN 3610 heeft voor een kleur gekozen. Wat zeggen jullie?